ecology.yaml

```diff
@@ -0,0 +1,525 @@
+
+  - name: principal_component_analysis
+    owner: devteam
+    tool_panel_section_label: 'Statistics'
+
+  - name: kernel_principal_component_analysis
+    owner: devteam
+    tool_panel_section_label: 'Statistics'
+
+  - name: kernel_canonical_correlation_analysis
+    owner: devteam
+    tool_panel_section_label: 'Statistics'
+
+  - name: canonical_correlation_analysis
+    owner: devteam
+    tool_panel_section_label: 'Statistics'
+
+  - name: sklearn_svm_classifier
+    owner: bgruening
+    tool_panel_section_label: 'Statistics'
+
+  - name: sklearn_sample_generator
+    owner: bgruening
+    tool_panel_section_label: 'Statistics'
+
+  - name: sklearn_searchcv
+    owner: bgruening
+    tool_panel_section_label: 'Statistics'
+
+  - name: sklearn_build_pipeline
+    owner: bgruening
+    tool_panel_section_label: 'Statistics'
+
+  - name: sklearn_pairwise_metrics
+    owner: bgruening
+    tool_panel_section_label: 'Statistics'
+
+  - name: sklearn_nn_classifier
+    owner: bgruening
+    tool_panel_section_label: 'Statistics'
+
+  - name: sklearn_numeric_clustering
+    owner: bgruening
+    tool_panel_section_label: 'Statistics'
+
+  - name: sklearn_ensemble
+    owner: bgruening
+    tool_panel_section_label: 'Statistics'
+
+  - name: sklearn_feature_selection
+    owner: bgruening
+    tool_panel_section_label: 'Statistics'
+
+  - name: sklearn_data_preprocess
+    owner: bgruening
+    tool_panel_section_label: 'Statistics'
+
+  - name: sklearn_clf_metrics
+    owner: bgruening
+    tool_panel_section_label: 'Statistics'
+
+  - name: sklearn_regression_metrics
+    owner: bgruening
+    tool_panel_section_label: 'Statistics'
+
+  - name: sklearn_discriminant_classifier
+    owner: bgruening
+    tool_panel_section_label: 'Statistics'
+
+  - name: sklearn_generalized_linear
+    owner: bgruening
+    tool_panel_section_label: 'Statistics'
+
+  - name: sklearn_model_validation
+    owner: bgruening
+    tool_panel_section_label: 'Statistics'
+
+  - name: scipy_sparse
+    owner: bgruening
+    tool_panel_section_label: 'Statistics'
+
+  - name: sklearn_suite
+    owner: bgruening
+    tool_panel_section_label: 'Statistics'
+
+
+  - name: regex_find_replace
+    owner: galaxyp
+    tool_panel_section_label: 'Text Manipulation'
+
+
+  - name: suite_obitools
+    owner: iuc
+    tool_panel_section_label: 'DNA metabarcoding'
+    
+
+  - name: fastq_groomer
+    owner: devteam
+    tool_panel_section_label: 'Quality check'
+    
+  - name: fastqc
+    owner: devteam
+    tool_panel_section_label: 'Quality check'
+    
+  - name: multiqc
+    owner: iuc
+    tool_panel_section_label: 'Quality check'
+    
+  - name: mothur_sffinfo
+    owner: iuc
+    tool_panel_section_label: 'File Conversion'
+    
+  - name: fastq_combiner
+    owner: devteam
+    tool_panel_section_label: 'File Conversion'
+    
+  - name: bwa
+    owner: devteam
+    tool_panel_section_label: 'Building LOCI from RADseq'
+
+  - name: suite_stacks
```

Comment:
Can you please list all tools of this suite and not the suite alone.
Please note that adding those here is not strictly needed as we do have those tools already installed.

Reply:
Done! In this list I put all eology related tools but for sure some/ a lot are already on usegalaxy.eu. Is it ok for you or you prefer an "exclusion list" ?